src/templates/specific-project.js

```diff
@@ -22,11 +24,27 @@ const ProjectTemplate = ({ data }) => (
                 <Link style={{color: '#1C2833', paddingBottom: '20px'}} to={`/${data.strapiProject.profiles.profile_name}`}>Sponsored by {data.strapiProject.profiles.profile_name}</Link>
                 <br/>
                 <br/>
-                <Link className="button has-text-centered" href='/' type="default" block>
-                  Contact Sponsor
-                </Link>
+                <br/>
+              <div className="columns is-centered is-flex" >
+                <div className="column is-one-quarter is-flex">
+                </div>
+                  {data.strapiProject.project_github !== "" &&
+                   <div className="column is-centered is-flex">                   
+                    <a href={data.strapiProject.project_github}><FaGithub size={"4.5em"}/></a></div>}
```

Comment:
The icons are too big - not just because @liliedelamotte says so, but because they are causing overlap issues when resizing the page.